src/com/houarizegai/calculator/Calculator.java

```diff
@@ -483,7 +486,7 @@ private Calculator() {
         });
         window.add(btnEqual);
 
-        btnRoot = new JButton("√");
```

Comment:
in my experience, this one is really tricky, sometimes it's working sometimes not